doc/source/examples/ale_classification.ipynb

```diff
@@ -23,7 +23,7 @@
   },
```

Comment:
<p>result into a decrease -&gt; result in a decrease?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/846/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>